client/components/domains/example-domain-suggestions/index.jsx

```diff
@@ -92,7 +93,7 @@ module.exports = React.createClass( {
 					},
 
 					components: {
-						mappingLink: <a onClick={ this.handleClickMappingLink } href={ mappingUrl } />,
+						mappingLink: <a onClick={ this.handleClickMappingLink } href="#" />,
```

Comment:
We need to pass it down through props because it changes depending on whether this is mounted in signup.